test/integration/main_test.go

```diff
@@ -0,0 +1,56 @@
+package integration
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func KubeClient() (result *kubernetes.Clientset) {
+	kubeconfig := os.Getenv("KUBECONFIG") // variable is a path to the local kubeconfig
```

Comment:
would be good to test whether the `$KUBECONFIG` exists at all

Reply:
done using `os.LookupEnv`